### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] enable testing of released packages in Travis CI matrix
- [ ] bump minimal requirements of `invenio-*` dependencies

**Comment by SamiHiltunen** [comment]
@jirikuncar enabled release package testing and bumped up the dependency versions

**Comment by jirikuncar** [comment]
- [ ] amend commit messages `s/add/fix for/`

**Comment by SamiHiltunen** [comment]
@jirikuncar amended commit messages

File: tests/test_collection_filter_enhancer.py [hunk 1]
```diff
@@ -19,12 +19,13 @@
...
 """Test policies on restricted and permitted collections."""
 
-from invenio_search.enhancers.collection_filter import \
-    create_collection_query
-from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
-
 from invenio_query_parser.ast import AndOp, NotOp, OrOp
 
+from invenio_search.enhancers.collection_filter import create_collection_query
+
```
Comment (jirikuncar) [inline tests/test_collection_filter_enhancer.py]: too many empty lines?

File: tests/test_elasticsearch_walker.py [hunk 2]
```diff
@@ -43,14 +45,14 @@ def setUp(self):
     def test_empty_query(self):
         self.assertEqual(EmptyQuery("").accept(self.converter), {
             "match_all": {
-                }})
+            }})
```
Comment (jirikuncar) [inline tests/test_elasticsearch_walker.py]:
```python
self.assertEqual(EmptyQuery("").accept(self.converter), {
    "match_all": {}
})
```
wha

**Pull request merged**